gdrive

```diff
@@ -100,37 +105,45 @@ def list_files(service, file_name, list_all=False):
     while next_page is not None:
         if input(">>> Press Enter for the next results\n"):
             break
-        files, next_page = search_filename(service, file_name, 10, next_page)
+        files, next_page = search_filename(service, file_name, 10, list_all, next_page)
         for file in files:
             describe_file(file)
 
 def describe_file(file):
```

Comment:
You're actually receiving file metadata as parameter and not file. Maybe rename this 'file' variable name since you're already working on this method.

Reply:
Changed to 'file' to 'metadata' and 'files' to 'files_found'